src/App.js

```diff
@@ -60,23 +65,42 @@ class App extends Component {
       filteredPoiByKeyword,
       barsAreDisplayed,
       displaySecondImportancePoiPage,
+      poiSampleDisplay,
     } = this.props;
     return (
       <div>
-        {barsAreDisplayed && <SearchBar />}
-        { isFirstResearchDone
+        {!poiSampleDisplay.length ? (
+          <div className="loadingPage">
+            <p>
+Attention mesdames et messieurs,
+              <br />
+          dans un instant,
+              <br />
+          ça va s'afficher...
+            </p>
+            <div className="loader" />
+          </div>
+        )
+          : (
+
+            <div>
+              {barsAreDisplayed && <SearchBar />}
+              { isFirstResearchDone
         && !filteredPoiByKeyword.length
         && <Alert /> }
 
-        <AppMap />
-        {Object.keys(specificPoiInfos).length && <PoiInformation />}
-        {!Object.keys(specificPoiInfos).length && barsAreDisplayed && <FilterBar />}
-        {isCreateFormDisplayed && <CreatePoiForm />}
-        {filterKeywordPageDisplay && <SelectCategorie />}
-        {displaySecondImportancePoiPage && <SelectSecondImportancePoi />}
+              <AppMap />
+              {Object.keys(specificPoiInfos).length && <PoiInformation />}
+              {!Object.keys(specificPoiInfos).length && barsAreDisplayed && <FilterBar />}
```

Comment:
Ternary condition instead for raws 93 and 94

Reply:
ok